.github/workflows/test-gha-eks.yml

```diff
@@ -0,0 +1,182 @@
+---
+name: EKS Cluster with an AuroraDB creation and destruction test
+
+on:
+  schedule:
+    - cron: '0 1 * * 1' # At 01:00 on Monday.
```

Comment:
let's maybe go with Tuesday.
I feel like we're piling up too much on Mondays at the same time already and this way we it ensure the Renovate PRs merged on Monday do no harm.